blueprints/custom-post/blueprints.json

```
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+    "meta": {
+        "title": "Custom Post Type: Books",
+        "description": "Blueprint that added a custom post type to playground",
+        "author": "bph",
+        "categories": ["Content", "CPT"]
+    },
+    "landingPage": "/wp-admin/",
+    "steps":[
+            {
+                "step": "login"
+            },
+            {
+                "step": "mkdir",
+                "path": "/wordpress/wp-content/plugins/books"
+            },
+            {
+                "step": "writeFile",
+                "path": "/wordpress/wp-content/plugins/books/books.php",
+                "data": {
+                    "resource": "url",
+                    "url": "https://raw.githubusercontent.com/adamziel/blueprints/custom-post/book.txt"
```

Nitpick: Branch name is required in these URLs:

```suggestion
                    "url": "https://raw.githubusercontent.com/adamziel/blueprints/trunk/custom-post/book.txt"
```

Note the URL won't work until this PR is merged. [This issue](https://github.com/adamziel/blueprints/issues/6) is about finding a nice solution to this URL problem.